.github/workflows/integration.yml

```diff
@@ -89,6 +89,27 @@ jobs:
           serve -m test::PyTorch=file:///opt/ml/model/resnet18_serving_all.zip?model_name=resnet18
           ./test_client.sh http://127.0.0.1:8080/predictions/test image/jpg https://resources.djl.ai/images/kitten.jpg
           docker rm -f $(docker ps -aq)
+      - name: Test MxNet binary mode
+        working-directory: tests/integration
+        run: |
+          ./launch_container.sh deepjavalibrary/djl-serving:${{ matrix.arch }}-nightly $PWD/models ${{ matrix.arch }} \
+          serve -m test::MXNet=file:///opt/ml/model/ssd_resnet50.zip?model_name=ssd_resnet50
+          ./test_client.sh http://127.0.0.1:8080/predictions/test image/jpg https://resources.djl.ai/images/kitten.jpg
+          docker rm -f $(docker ps -aq)
+      - name: Test ONNX binary mode
+        working-directory: tests/integration
+        run: |
+          ./launch_container.sh deepjavalibrary/djl-serving:${{ matrix.arch }}-nightly $PWD/models ${{ matrix.arch }} \
+          serve -m test::OnnxRuntime=file:///opt/ml/model/resnet18-v1-7.zip?model_name=resnet18-v1-7
+          ./test_client.sh http://127.0.0.1:8080/predictions/test image/jpg https://resources.djl.ai/images/kitten.jpg
+          docker rm -f $(docker ps -aq)
+      - name: Test Tensorflow binary mode
+        working-directory: tests/integration
+        run: |
+          ./launch_container.sh deepjavalibrary/djl-serving:${{ matrix.arch }}-nightly $PWD/models ${{ matrix.arch }} \
+          serve -m test::TensorFlow=file:///opt/ml/model/resnet50v1.zip?model_name=resnet50
+          ./test_client.sh http://127.0.0.1:8080/predictions/test tensor/ndlist 1,3,224,224
```

Comment:
make shape to 1,224,224,3 to match NHWC (Number of images, heigth, width, channel) format required by tensorflow.